### ISSUE TITLE
Mistake in Basic.v

### ISSUE THREAD

**Issue by Sigmoni** [Ailrun/software_foundations_solution#3]
The answer:
```
Fail Fixpoint plus'' (n : nat) (m : nat) : nat :=
  match n with
  | O => m
  | S n' => S (plus'' m n)
  end.
```
Does not satisfy the requirement that it terminates eventually.
I suppose it should be:
```
Fail Fixpoint plus'' (n : nat) (m : nat) : nat :=
  match n with
  | O => m
  | S n' => S (plus'' m n')
  end.
```

**Comment by Ailrun** [comment]
True, I wrote `| S n => ...` first but refactored it to the new name, but while refactoring it missed the later one.

**Comment by Ailrun** [comment]
Fixed in 4a8f277051a7c847869b3f44e5a639622c561db2

**Comment by Ailrun** [comment]
if
